pkg/utils/openshift/utils.go

```diff
@@ -1,45 +1,24 @@
 package openshift
 
 import (
-	"k8s.io/client-go/discovery"
+	"github.com/jeremyary/operator-utils/pkg/platform"
```

Comment:
should be `github.com/RHsyseng/operator-utils/pkg/platform`

Reply:
Updated my local enviro & import corrected per our chat. Thanks for pointers!